CHANGELOG.md

```diff
@@ -1,224 +1,228 @@
+## 0.0.25-dev0
+
+### Fixes
+
+- **Fix Notion Ingestion** Fix the Notion source connector to work with the latest version of the Notion API (added `in_trash` properties to `Page`, `Block` and `Database`).
+
 ## 0.0.25
 
 ### Enhancements
 
-* **Support pinecone namespace on upload**
-* **Migrate Outlook Source Connector to V2**
-* **Support for Databricks Volumes source connector**
+- **Support pinecone namespace on upload**
+- **Migrate Outlook Source Connector to V2**
+- **Support for Databricks Volumes source connector**
 
 ### Fixes
 
-* **Update Sharepoint Creds and Expected docs**
+- **Update Sharepoint Creds and Expected docs**
```

Comment:
Looks like something updated all of these, can we limit the changes to the changelog in this PR?